File: Anul II/Medii de proiectare si programare/Laborator/lab09/networking/src/main/java/com/ubb/protocol/request/CreateRaceEntriesRequest.java
package com.ubb.protocol.request;

import com.ubb.model.RaceEntry;

import java.util.List;

public class CreateRaceEntriesRequest implements Request {
    private final List<RaceEntry> raceEntries;

    public CreateRaceEntriesRequest(List<RaceEntry> raceEntries) {
        this.raceEntries = raceEntries;
    }

    public List<RaceEntry> getRaceEntries() {
        return raceEntries;
    }
}

File: Anul II/Medii de proiectare si programare/Laborator/lab08/C#/Ubb.BikeContest.Networking/Protocol/Request/CreateRaceEntriesRequest.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Ubb.BikeContest.Model;

namespace Ubb.BikeContest.Networking.Protocol.Request
{
    [Serializable]
    public class CreateRaceEntriesRequest : IRequest
    {
        private readonly List<RaceEntry> raceEntries;

        public CreateRaceEntriesRequest(List<RaceEntry> raceEntries)
        {
            this.raceEntries = raceEntries;
        }

        public virtual List<RaceEntry> RaceEntries
        {
            get { return raceEntries; }
        }
    }
}
